### ISSUE TITLE
Empty arrays in annotations make the following annotations come as children

### ISSUE THREAD

**Issue by Mouradif** [glayzzle/doc-parser#7]
Hello there again,

I noticed a very similar problem as with issue #4 : this time it's with empty arrays.

Example :

This comment : 
```
/**
 * @var string[]
 *
 * @SomethingElse()
 */
```

is currently parsed as :

```
{
  "kind": "doc",
  "summary": "",
  "body": [
    {
      "kind": "block",
      "name": "var",
      "options": [
        {
          "kind": "word",
          "value": "Channel"
        },
        {
          "kind": "array",
          "value": [
            {
              "kind": "annotation",
              "name": "somethingelse",
              "arguments": []
            }
          ]
        }
      ]
    }
  ]
}
```

**Comment by Mouradif** [comment]
Hello,

Any comment on that PR #8 ? Something I should change ?

**Comment by ichiriac** [comment]
Hi @Mouradif,

Thanks for the PR - I've released your fix under 0.4.9
